app/models/institutions_archive.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require_dependency 'institution'
+
+class InstitutionsArchive < Institution
+  self.table_name = 'institutions_archives'
+
+  # class methods
+  def self.archive(version)
```

Comment:
how long will this method take to run the first time it's run and what will be the results to the app while it's running.  It seems like inserting and deleting 5million rows will be problematic.

Reply:
<img width="476" alt="Screen Shot 2019-07-09 at 7 19 55 AM" src="https://user-images.githubusercontent.com/1094999/60884070-021fd200-a21a-11e9-9c28-b2ca276b532e.png">

From https://dsva.slack.com/archives/CBU0KDSB1/p1562596229148600

Reply:
I'm not sure exactly when the explain was running, but around that time yesterday  the GIDS cpu was pegged at 100% <img width="1467" alt="Screen Shot 2019-07-09 at 10 55 21 AM" src="https://user-images.githubusercontent.com/19188/60899794-17582900-a239-11e9-9f98-57b0f089b77f.png">

Reply:
and we were unable to fufill requests.  Not sure how much of that was due to a traffic spike or what.
![image](https://user-images.githubusercontent.com/19188/60900344-08be4180-a23a-11e9-8123-3c6d150f3098.png)

Reply:
I believe this issue was due to autocomplete causing issues, I think @jholton can speak to this

Reply:
Yeah, my `EXPLAIN` queries should have put no load on the database.  It looks like our spike was due to `autocomplete` getting hammered.